src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -196,4 +200,154 @@ Scalar<DataType> time_deriv_of_lapse(
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::iJ<DataType, SpatialDim, Frame> spatial_deriv_of_shift(
+    const Scalar<DataType>& lapse,
+    const tnsr::AA<DataType, SpatialDim, Frame>& inverse_spacetime_metric,
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes time derivative of the shift vector from
+ *        the generalized harmonic and geometric variables
+ *
+ * \details Let the generalized harmonic spatial derivative variable be
+ * \f$\Phi_{iab} = \partial_i \psi_{ab} \f$. The time derivative of \f$ N^i \f$
+ * is given by:
+ * \f{align*}
+ *  -N^2 t^b \Pi_{kb} g^{ik} + N N^j t^b \Phi_{jkb} g^{ik}
+ *  =& N g^{ik} t^b (\partial_0 - N^j\partial_j) \psi_{kb}
+ *                  + N g^{ik} t^b N^j\partial_j \psi_{kb} \\
+ *  =& N g^{ik} t^b \partial_0 \psi_{kb} \\
+ *  =& g^{ik} \partial_0 (g_{kj} N^j) - N^j g^{ik} \partial_0 g_{kj} \\
+ *  =& \partial_0 N^i
+ * \f}
+ * where the second to last to last line used the same manipulations
+ * as at the end of the spatial derivative calculation. Therefore:
+ * \f{align*}
+ * \partial_0 N^i =& -N^2 t^b\Pi_{kb} g^{ik} + N N^j t^b\Phi_{jkb} g^{ik} \\
+ *                =& -N g^{ik} t^b (N \Pi_{kb} - N^j \Phi_{jkb})
+ * \f}
+ */
+template <size_t SpatialDim, typename Frame, typename DataType>
+void time_deriv_of_shift(
+    gsl::not_null<tnsr::I<DataType, SpatialDim, Frame>*> dt_shift,
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept;
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::I<DataType, SpatialDim, Frame> time_deriv_of_shift(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes time derivative of index lowered shift from generalized
+ *        harmonic variables, spatial metric and its time derivative.
+ *
+ * \details The time derivative of \f$ N_i \f$ is given by:
+ * \f{align*}
+ *  \partial_0 N_i = g_{ij} \partial_0 N^j + N^j \partial_0 g_{ij}
+ * \f}
+ * where the first term is obtained from `time_deriv_of_shift`, and the latter
```

Comment:
`time_deriv_of_shift` -> `time_deriv_of_shift()` (so doxygen generates the link to the function)

Reply:
changed